src/main/java/com/redi/shortener/controller/TimeController.java

```diff
@@ -0,0 +1,25 @@
+package com.redi.shortener.controller;
+
+import com.redi.shortener.model.CreateTimeRequest;
+import com.redi.shortener.services.TimeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@RestController
+public class TimeController {
+    @Autowired TimeService service;
+
+    @GetMapping("/time")
+    public static String dateAndTime(@RequestBody final CreateTimeRequest request){
```

Comment:
Great, you made it 🎉

Just a small commend: it seems that you implemented the logic of getting time in the controller, so you don't need `TimeService`